kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/utils/FieldNameSanitizerTest.java

```diff
@@ -0,0 +1,45 @@
+package com.wepay.kafka.connect.bigquery.utils;
+
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class FieldNameSanitizerTest {
+  HashMap testMap;
```

Comment:
since you only use it in one test method, you should declare it inside the test.
If you plan to use the same map across multiple test methods, then you can add a `setup()` method annotated with `@Before` to automatically initialize the map before every test method run.

Reply:
Solved!